include/zephyr/net/ieee802154.h

```diff
@@ -171,24 +205,23 @@ struct ieee802154_security_ctx {
 	 */
 	uint8_t key[16];
 
-	/** frame-level security key material */
+	/** Length in byptes of the interface-level security key material. */
 	uint8_t key_len;
 
 	/**
-	 * @brief security level
-	 *
-	 * @details Currently native L2 supports a single security level for all
-	 * frame types, commands and information elements, see section 9.4.2.2,
-	 * table 9-6 and ieee802154_security_level.
+	 * @brief Frame security level, possible values are defined in section
+	 * 9.4.2.2, table 9-6.
 	 *
-	 * @warning This is no longer in accordance with the current version of
-	 * the standard and needs to be extended in the future for full security
-	 * procedure compliance.
+	 * @warning Currently native L2 allows to configure one common security
+	 * level for all frame types, commands and information elements. This is
+	 * no longer in accordance with the current version of the standard and
```

Comment:
Here and a few other places: what is "current"??

Reply:
Good point. Specified the precise versions instead.